### ISSUE TITLE
Add doc/tags to gitignore

### ISSUE THREAD

**Issue by Jab2870** [junegunn/goyo.vim#154]
Hi, 
I use pathogen to manage all of my plugins which are git sub-modules. Unfortunately, as soon as I open vim, this plugin creates the file doc/tags which git notices as a new file and makes the sub-module dirty.

IF you could add the line 
`doc/tags`
to the .gitignore file, it would make life easier for anyone using pathogen

**Comment by Jab2870** [comment]
When I opened this, I didn't realise there was a pull request ( #153 ) open

**Comment by junegunn** [comment]
I can merge that, but many of my plugins also do not have .gitignore. Any reason you can't [configure your submodules to ignore changes](http://stackoverflow.com/questions/3240881/git-can-i-suppress-listing-of-modified-content-dirty-submodule-entries-in-sta)? It should be easier for you to do that than to ask every plugin maintainer.

**Comment by Jab2870** [comment]
Your plugin is the only plugin I use that has this issue.  I believe Pathogen (and vundle certainly used to although I don't use it now so can't be sure) create this file at run time. It is possible to ask git to ignore it but a lot of people use Pathogen.

Most other plugins I use have this .gitignore-d.

If you don't wish to ignore it, please could you add instructions to your readme to ignore dirty submodules?

Once the submodule has been added, you need to add a line to the `.gitmodules` file for your submodule.

Here is the relevant extract from my `.gitmodules` folder:

```
[submodule "bundle/goyo.vim"]
	path = bundle/goyo.vim
	url = https://github.com/junegunn/goyo.vim
	ignore = dirty
```

**Comment by junegunn** [comment]
Closed by #153
